packages/oasis-actions/src/types/ajna/AjnaPosition.ts

```diff
@@ -45,7 +45,16 @@ export class AjnaPosition implements IAjnaPosition {
   ) {}
 
   get liquidationPrice() {
-    return ZERO
+    const liquidationPrice = this.pool.mostOptimisticMatchingPrice
+      .times(
+        this.debtAmount
+          .div(this.pool.pendingInflator)
+          .times(this.pool.pendingInflator)
```

Comment:
Just to double check, Is this correct? I mean `div` by pending inflator and then `times`?

Reply:
Good question, but according to formulas in Notion it is:
`t0debt * pendingInflator`
and `t0debt` is `debtAmount / pendingInflator`

Reply:
Removed.